src/main/java/core/basesyntax/Application.java

```diff
@@ -2,6 +2,9 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
+        System.out.println(lottery.getRandomBall());
```

Comment:
![image](https://github.com/mate-academy/jv-lottery/assets/20326858/35d5247d-eb92-450e-a7e5-4b23a89264cf)

Reply:
In this case, I thought it was not necessary for the demonstration